source/documentation/deploying_apps/deploying_java.md

```diff
@@ -106,11 +106,15 @@ For more configuration options see [Tomcat Container](https://github.com/cloudfo
 
 ### Specify a Java version
 
-You should tell Cloud Foundry which version of Java your app uses in the Java buildpack.
+You can specify the version of the Java Runtime Environment (JRE) by setting the environment variable `JBP_CONFIG_OPEN_JDK_JRE`. For example to use JRE 11 you could add the following to your application's `manifest.yml`:
 
-Refer to the documentation on [buildpack language version updates](deploying_apps.html#buildpack-language-version-updates) for more information.
+```yaml
+  env:
+    JBP_CONFIG_OPEN_JDK_JRE: '{ jre: { version: 11.+ } }'
+ ```
+The `+` in the above example acts as a wildcard which will use the latest point release available.
```

Comment:
Suggest we say `version: 11.x` and remove line 115 as this should be known to the user.

Reply:
>  as this should be known to the user

I don't think it necessarily will be - this confused me when I was trying to get Java 11 running, I couldn't work out whether to use `+` or `x` or `*` or what.

Reply:
ok